Jenkinsfile

```diff
@@ -21,7 +21,7 @@ pipeline {
       steps {
         ansiColor('xterm') {
           // TODO: CO-77: add in scalafmt::test target
-          // sh "sbt ';headerCheck ;scalastyle ;scalafmt'"
+          // sh "sbt ';headerCheck ;scalastyle ;scalafmt::test'"
```

Comment:
Originally, because we are unable to enable the scalafmt plugin within our project code. CO-77 speculates that this is a Scala version thing?

However, I've just performed a version bump and it appears that all is good with the newer version of the scalafmt plugin. I'll version bump this and enable these Jenkins build settings.